src/bundle/Resources/views/content/tabs/content.html.twig

```diff
@@ -0,0 +1,22 @@
+{% set languageCode = content.versionInfo.contentInfo.mainLanguageCode %}
+
+{% for contentTypeGroup in contentType.contentTypeGroups %}
+    {% set contentTypeName = contentTypeGroup.name(languageCode) %}
+
+    <section class="ez-fieldgroups ez-fieldgroup ez-fieldgroup">
```

Comment:
too many classes from v1 here :)
For now, the following markup should be enough:
```html
<section class="ez-fieldgroup">
    <h3 class="ez-fieldgroup-name">{{ fieldgroupName }}</h3>
    <!-- fields here -->
</section>
```